framework_tvb/tvb/core/adapters/abcuploader.py

```diff
@@ -29,25 +29,40 @@
 #
 
 """
+.. moduleauthor:: Robert Vincze <robert.vincze@codemart.ro>
 .. moduleauthor:: Mihai Andrei <mihai.andrei@codemart.ro>
 """
 
 import os
 import numpy
 from abc import ABCMeta
+import pyAesCrypt
 from scipy import io as scipy_io
 from tvb.basic.logger.builder import get_logger
+from tvb.basic.profile import TvbProfile
 from tvb.core.adapters.abcadapter import ABCSynchronous, ABCAdapterForm
+from tvb.core.adapters.exceptions import LaunchException
 from tvb.core.entities.transient.structure_entities import DataTypeMetaData
-from tvb.core.neotraits.forms import StrField
+from tvb.core.neotraits.forms import StrField, TraitUploadField
 from tvb.core.neotraits.uploader_view_model import UploaderViewModel
+from cryptography.hazmat.primitives import serialization
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import padding
+from cryptography.hazmat.backends import default_backend
+from tvb.interfaces.command.demos.importers.encrypt_data import ENCRYPTED_DATA_SUFFIX, DECRYPTED_DATA_SUFFIX
```

Comment:
This dependency from `tvb.core` towards `tvb.interfaces` is not correct (it should be inverse).

Reply:
Yeah, I didn't pay attention to the dependencies. I reversed it now.